src/module/cp/cf.py

```
@@ -319,6 +323,11 @@ def accept_duel_pairing(message: RobotMessage, pair_code: str):
     message.reply("成功接受对战，正在进行随机选题")
 
     r_a, r_b = Codeforces.get_user_rating(user.handle), Codeforces.get_user_rating(opponent.handle)
+    if not r_a:
+        message.reply("你的账号状态异常，无法开始对战")
```

Ruff (0.13.1)

String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)